File: Pertemuan 4/programkalkulator.py
def penjumlahan(a, b):
    return a + b
  
def pengurangan(a, b):
    return a - b
  
def perkalian(a, b):
    return a * b
  
def divide(a, b):
    return a / b

def modulus(a, b):
    return a % b

print(":::::::::: Kalkulator Sederhana ::::::::::")
print()
print("1. Penjumlahan")
print("2. Pengurangan")
print("3. Perkalian")
print("4. Pembagian")
print("5. Modulus")
print()

pilihan = int(input("Masukkan pilihanmu : "))
print()
  

if pilihan == 1:
    print("----- Penjumlahan -----")
    print()
    nilaiA = int(input("Nilai a : "))
    nilaiB = int(input("Nilai b : "))
    print("Hasil penjumlahan dari", end=" ")
    print(nilaiA, "+", nilaiB, "=",
                    penjumlahan(nilaiA, nilaiB))
  
elif pilihan == 2:
    print("----- Pengurangan -----")
    print()
    nilaiA = int(input("Nilai a : "))
    nilaiB = int(input("Nilai b : "))
    print("Hasil pengurangan dari", end=" ")
    print(nilaiA, "-", nilaiB, "=",
                    pengurangan(nilaiA, nilaiB))
  
elif pilihan == 3:
    print("----- Perkalian -----")
    print()
    nilaiA = int(input("Nilai a : "))
    nilaiB = int(input("Nilai b : "))
    print("Hasil perkalian dari", end=" ")
    print(nilaiA, "*", nilaiB, "=",
                    perkalian(nilaiA, nilaiB))
  
elif pilihan == 4:
    print("----- Pembagian -----")
    print()
    nilaiA = int(input("Nilai a : "))
    nilaiB = int(input("Nilai b : "))
    print("Hasil pembagian dari", end=" ")
    print(nilaiA, "/", nilaiB, "=",
                    divide(nilaiA, nilaiB))

elif pilihan == 5:
    print("----- Modulus -----")
    print()
    nilaiA = int(input("Nilai a : "))
    nilaiB = int(input("Nilai b : "))
    print("Hasil modulus dari", end=" ")
    print(nilaiA, "%", nilaiB, "=",
                    modulus(nilaiA, nilaiB))

else:
    print("Invalid input")

print()
print(":::::::::::::::: Selesai ::::::::::::::::")

File: Pertemuan 4/programKalkulator.java
import java.util.Scanner;

public class programKalkulator {
    public static void main(String[] args){
        
        Scanner scanner = new Scanner (System.in);

        System.out.println(":::::::::: Kalkulator Sederhana ::::::::::");
        System.out.println();

        System.out.println("1. Penjumlahan");
        System.out.println("2. Pengurangan");
        System.out.println("3. Perkalian");
        System.out.println("4. Pembagian");
        System.out.println("5. Modulus");
        System.out.println();

        System.out.print("Masukkan Pilihanmu : ");
        int x = scanner.nextInt();
        switch (x) {
            case 1:
                Penjumlahan();
                break;
            case 2:
                Pengurangan();
                break;
            case 3:
                Perkalian();
                break;
            case 4:
                Pembagian();
                break;
            case 5:
                Modulus();
                break;
            
        }
        System.out.println();
        System.out.println(":::::::::::::::: Selesai ::::::::::::::::");
    }

    public static void Penjumlahan() {
        Scanner scanner = new Scanner (System.in);
        System.out.println();
        System.out.println("----- Penjumlahan -----");
        System.out.println();
        
        System.out.print("Nilai a = ");
        int a = scanner.nextInt();
        System.out.print("Nilai b = ");
        int b = scanner.nextInt();
        int hasil = a + b;
        System.out.println();
        System.out.println("Hasil penjumlahan dari " + a + " + " + b + " = " + hasil);
    }
    
    public static void Pengurangan() {
        Scanner scanner = new Scanner (System.in);
        System.out.println();
        System.out.println("----- Pengurangan -----");
        System.out.println();
        
        System.out.print("Nilai a = ");
        int a = scanner.nextInt();
        System.out.print("Nilai b = ");
        int b = scanner.nextInt();
        int hasil = a - b;
        System.out.println();
        System.out.println("Hasil pengurangan dari " + a + " - " + b + " = " + hasil);
    }
    
    public static void Perkalian() {
        Scanner scanner = new Scanner (System.in);
        System.out.println();
        System.out.println("----- Perkalian -----");
        System.out.println();
        
        System.out.print("Nilai a = ");
        int a = scanner.nextInt();
        System.out.print("Nilai b = ");
        int b = scanner.nextInt();
        int hasil = a * b;
        System.out.println();
        System.out.println("Hasil perkalian dari " + a + " * " + b + " = " + hasil);
    }
    
    public static void Pembagian() {
        Scanner scanner = new Scanner (System.in);
        System.out.println();
        System.out.println("----- Pembagian -----");
        System.out.println();
        
        System.out.print("Nilai a = ");
        int a = scanner.nextInt();
        System.out.print("Nilai b = ");
        int b = scanner.nextInt();
        int hasil = a / b;
        System.out.println();
        System.out.println("Hasil pembagian dari " + a + " / " + b + " = " + hasil);
    }
    public static void Modulus() {
        Scanner scanner = new Scanner (System.in);
        System.out.println();
        System.out.println("----- Modulus -----");
        System.out.println();
        
        System.out.print("Nilai a = ");
        int a = scanner.nextInt();
        System.out.print("Nilai b = ");
        int b = scanner.nextInt();
        int hasil = a % b;
        System.out.println();
        System.out.println("Hasil modulus dari " + a + " % " + b + " = " + hasil);
    }
}
